inbm/dispatcher-agent/dispatcher/dispatcher_class.py

```diff
@@ -782,12 +782,13 @@ def process_scheduled_requests(scheduled_requests: Sequence[Schedule]):
             logger.debug(f"Scheduled repeated job: {repeated_schedule}")
 
         for imm in schedule.immedate_requests:
+            dispatcher._send_result("", request_id)
```

Comment:
we should only send one result per request, not multiple. if all we have left are immediate requests, we should send one result with blank error and the request id first, and then process the requests.

Reply:
I have moved the _send_result out of the for loop. The dispatcher will send back the acknowledgement response one time before processing the immediate scheduling.